extensions/proposals/WEBGL_video_texture/extension.xml

```diff
@@ -53,10 +60,10 @@ interface WebGLVideoFrameInfo {
 
 [NoInterfaceObject]
 interface WEBGL_video_texture {
-    const GLenum TEXTURE_VIDEO_IMAGE             = 0x851D;
-    const GLenum SAMPLER_VIDEO_IMAGE             = 0x8B61;
+    const GLenum TEXTURE_VIDEO_IMAGE_WEBGL             = 0x851D;
+    const GLenum SAMPLER_VIDEO_IMAGE_WEBGL             = 0x8B61;
 
-    [RaisesException] WebGLVideoFrameInfo VideoElementTargetVideoTexture(
+    [RaisesException] WebGLVideoFrameInfo shareVideoImageWEBGL(
```

Comment:
Our group's tech lead and I had a conversation about this extension last week. Sharing the video texture to WebGL conceptually "locks" the video frame, at least in Chrome's media stack, and it's important to limit the duration that the frame might be locked.

Could you please add a new operation like "releaseVideoImageWEBGL" which takes the GLenum target, and releases the video frame from the bound texture?

Could you please also add some specification text like:

```
If the application does not call releaseVideoImageWEBGL, then all of the WebGLTextures
with which video frames were shared will be implicitly released when the current
JavaScript code returns control to the user agent.
```

It's important to make it clear to application developers that they are supposed to do all of their rendering work in the same callback where they acquire those video frames.

Thanks for considering these changes.

Reply:
Sure, it makes sense. Thanks for your reviewing !

Reply:
I try to add the new API and update the example. PTAL if you have time!